mycli/main.py

```diff
@@ -89,7 +89,7 @@ class MyCli(object):
     ]
 
     default_config_file = os.path.join(PACKAGE_ROOT, 'myclirc')
-
+    pwd_config_file = os.path.join(os.getcwd(), "")
```

Comment:
I think you'll need to join the cwd and ".myclirc" instead of an empty string...